docker-compose.yml

```diff
@@ -77,7 +61,11 @@ networks:
 
 ####################### Volumes Definition #######################
 volumes:
-  pims-database-data:
-    name: pims-database-data
+  database-data:
```

Comment:
I believe you can simplify this by just placing the path on the left side on line 15.

Reply:
Yes you are correct, but I opted for the named volume bind mount because its clear what the volume is used for in Docker Desktop and if there is a need to use any commands involving the volume, we have the name of the volume we can use (such as in the make clean command where it removes docker volumes).